### ISSUE TITLE
twitterImg.url incorrect

### ISSUE THREAD

**Issue by ktrzeciaknubisa** [jshemas/openGraphScraper#30]
So when I try to scrape https://github.com, I get:

```js
"twitterImage": {
            "url": "1200",
            "width": "1200"
        }
```

whereas:

```html
      <meta property="twitter:image:src" content="https://assets-cdn.github.com/images/modules/open_graph/github-logo.png">
      <meta property="twitter:image:width" content="1200">
      <meta property="twitter:image:height" content="1200">
```

I see the following problems:
1. `twitter:image:height` did not get scraped
2. probably `twitter:image:src` got scraped instead of `twitter:image:height`
3. why there is `twitterImage.url` rather than `twitterImage.src` which would be more consistent to `twitter:image:src` (there is no `twitter:image:url` after all)?

**Comment by pyrotechnick** [comment]
data.twitterImage.* and others fail for many sites including https://atom.io

The problem arises from the arbitrarily-ordered items defined by [`mediaMapperTwitterImage`](https://github.com/jshemas/openGraphScraper/blob/c56425016a4ef119683f4a9c276aa67837d6a3ee/app.js#L255)

**Comment by jshemas** [comment]
Hello,

This has been fix in `open-graph-scraper@2.3.6`! Thanks for pointing it out! @pyrotechnick was right about the issue, so I added in some default values. 

To answer number question number three, it's simple `url` because the rest of the object we send back uses `url` over `src`. I wanted to keep things consistent.
